dotCMS/src/main/java/com/dotmarketing/cms/forgotpassword/action/ForgotPasswordAction.java

```diff
@@ -48,7 +49,8 @@
  * @version $Revision: 1.5 $
  *
  */
-public class ForgotPasswordAction extends DispatchAction {
+@Deprecated
+public class ForgotPasswordAction extends SecureAction {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)